docs/tutorials/developer-liftoff/level-3/3.5-identities-and-auth.mdx

```diff
@@ -92,81 +92,12 @@ Now let's look at a simple example of how to integrate Internet Identity into th
 
 Open the [ICP Ninja Who am I? example](https://icp.ninja/projects/who-am-i). In the project's `dfx.json` file, you will see a definition for the Internet Identity canister:
 
-```json title="dfx.json"
-...
-    "internet_identity": {
-      "candid": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity.did",
-      "type": "custom",
-      "specified_id": "rdmx6-jaaaa-aaaaa-aaadq-cai",
-      "remote": {
-        "id": {
-          "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
-        }
-      },
-      "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm.gz"
-    }
-...
+```json title="dfx.json" file=../../../references/samples/motoko/who_am_i/dfx.json
```

Comment:
can we also reference certain lines here or is it on purpose to show the complete `dfx.json` ?